README.md

```diff
@@ -2,3 +2,62 @@ Msgstat
 ----------
 Msgstat provides a CLI tool which analyses status-go log files and generates corresponding output of information 
 related to messages delivered over whisper (either rpc or p2p).
+
+## Commands
+
+### `-readfile`
+
+Msgstat `-readfile` expects to be provided a path to the proper log file which will be processed to generate reports.
+
+```bash
+> msgstat readfile ./logs/status-im/status-go/11.21.2017.log
+```
+
+### `-read`
+
+Msgstat `-read` expects to be data from a log file written into `stdin` which will be processed to generate reports.
+
+```bash
+> cat ./logs/status-im/status-go/11.21.2017.log | msgstat read
+```
+
+## Report Format
+
+Msgstat currently outputs parsed message into the following format:
+
+```
+Envelope #4T0NEM0Qrcm1uVVhBVmhYYW9HS1AwQ01xWmhwQmJGSDBnOHZpWnN2
```

Comment:
Looks good! I would just change the output to some known format like JSON or YAML so that it can be reused with other tools.

Reply:
Am wondering if we should output individual `json` values or a single big json that contains said details from the log file, your thoughts ?

Reply:
It's a good question. Not really sure. Go makes it easy to output a single, valid JSON blob. I guess it should play better with other tools.

Reply:
Which sizes do we expect? JSON via marshalling could be a huge object in memory. Maybe better something like a CSV to iterate over. And it's also more simple to import into standard databases for manual queries and analysis there.

Reply:
Hmnmm, this is indeed an interesting question.

Reply:
But Go supports streamable JSON marshaling. It shouldn't require putting the whole object into memory.